src/main.rs

```diff
@@ -378,19 +378,19 @@ mod test {
             }
         }
 
-        fn check_service_compatible(
+        fn check_service_equal(
             new_name: &str,
             new: candid_parser::utils::CandidSource,
             old_name: &str,
             old: candid_parser::utils::CandidSource,
         ) {
             let new_str = source_to_str(&new);
             let old_str = source_to_str(&old);
-            match candid_parser::utils::service_compatible(new, old) {
```

Comment:
@THLO @rvanasa 
I realized that this was not sufficient when starting working on #308, where I added the Rust skeleton for a new endpoint but nothing in the Candid file and the test was still green. Having them equal is what we want IMO and this picked up a few inconsistencies from past MRS, where the Candid file was not changed appropriately.